bftengine/src/bftengine/SimpleClientImp.cpp

```diff
@@ -419,13 +419,22 @@ void SimpleClientImp::sendPendingRequest() {
   }
 }
 
+// SeqNumberGeneratorForClientRequestsImp, generates unique, monotonically increasing, sequence number.
+// The SN is a time stamp processed and derived from the system clock.
+// The SN is a bitwise or between lastMilli and lastCount.
+// Equal time stamp will be bitwise or, with different lastCount values,
+// and uniqueness will bre preserved.
 class SeqNumberGeneratorForClientRequestsImp : public SeqNumberGeneratorForClientRequests {
   virtual uint64_t generateUniqueSequenceNumberForRequest() override;
   virtual uint64_t generateUniqueSequenceNumberForRequest(
       std::chrono::time_point<std::chrono::system_clock> now) override;
 
  protected:
+    // limited to the size lastMilli shifted.
+    const u_int64_t lastCountLimit = 0x3FFFFF;
```

Comment:
Constants are usually expressed in capital letters, separated by underscores. See examples in ReplicaImp for example.